docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -709,6 +709,37 @@ Key interfaces
         :raises cryptography.exceptions.InvalidSignature: If the signature does
             not validate.
 
+    .. method:: recover(signature, padding, algorithm)
+
+        .. versionadded:: 3.3
+
+        Recovers the digest of the original message string from the signature.
+        Normally you should use the
+        :meth:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey.verify`
+        function to validate the signature. But there are some cases where you
+        may want to recover the digest of the original message.
```

Comment:
I believe you mentioned some specific protocols that expect to be able to recover the digest in the original issue -- could we elaborate a tiny bit here and add an example or two of where it's necessary?

Reply:
Fixed. Added a few examples to the documentation.